[CI] DataTiersUsageTransportActionTests testCalculateMAD failures

**Build scan**:
https://gradle-enterprise.elastic.co/s/n5hni2kkvpfpu/console-log?task=:x-pack:plugin:core:test
https://gradle-enterprise.elastic.co/s/w7ivqkxh4pyao/console-log?task=:x-pack:plugin:core:test

**Repro line**:
./gradlew ':x-pack:plugin:core:test' --tests "org.elasticsearch.xpack.core.DataTiersFeatureSetTests.testCalculateMAD" \
  -Dtests.seed=991EA0DD80DF089C \
  -Dtests.security.manager=true \
  -Dtests.locale=en \
  -Dtests.timezone=BET \
  -Druntime.java=11

**Reproduces locally?**:
No

**Applicable branches**:
master, 7.10

**Failure history**:
Not many.

https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-30d,mode:quick,to:now))&_a=(columns:!(build.branch,build.build-scans),index:b646ed00-7efc-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:testCalculateMAD),sort:!(process.time-start,desc))

**Failure excerpt**:
```
org.elasticsearch.xpack.core.DataTiersUsageTransportActionTests > testCalculateMAD FAILED   
    java.lang.AssertionError:   
    Expected: <1L>  
         but: was <0L>  
        at __randomizedtesting.SeedInfo.seed([1E8998B9CCE4D61E:AB7675CAA19F5D92]:0) 
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18) 
        at org.junit.Assert.assertThat(Assert.java:956) 
        at org.junit.Assert.assertThat(Assert.java:923) 
        at org.elasticsearch.xpack.core.DataTiersUsageTransportActionTests.testCalculateMAD(DataTiersUsageTransportActionTests.java:40) 
```

--- a/x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/DataTiersUsageTransportActionTests.java
+++ b/x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/DataTiersUsageTransportActionTests.java
@@ -26,10 +26,11 @@ import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class DataTiersUsageTransportActionTests extends ESTestCase {
+
     public void testCalculateMAD() {
         assertThat(DataTiersUsageTransportAction.computeMedianAbsoluteDeviation(new TDigestState(10)), equalTo(0L));
 
-        TDigestState sketch = new TDigestState(randomDoubleBetween(0, 1000, false));
+        TDigestState sketch = new TDigestState(randomDoubleBetween(1, 1000, false));
         sketch.add(1);
         sketch.add(1);
         sketch.add(2);

